scenariolib/event_click.go

```diff
@@ -34,7 +34,7 @@ func newClickEvent(e *JSONEvent) (*ClickEvent, error) {
 		return nil, errors.New("Parameter probability must be a number between 0 and 1 in a ClickEvent")
 	}
 
-	if docNo, validcast = e.Arguments["offset"].(float64); !validcast {
```

Comment:
My bad I changed those without re-doing all the tests just to change how the code was with the validCasts and all, probably a copy paste error.